vulnfeeds/test_data/nvdcve-2.0/CVE-2024-47177.json

```diff
@@ -0,0 +1 @@
+{"resultsPerPage":1,"startIndex":0,"totalResults":1,"format":"NVD_CVE","version":"2.0","timestamp":"2024-10-17T03:00:48.680","vulnerabilities":[{"cve":{"id":"CVE-2024-47177","sourceIdentifier":"security-advisories@github.com","published":"2024-09-26T22:15:04.740","lastModified":"2024-09-30T12:46:20.237","vulnStatus":"Awaiting Analysis","cveTags":[],"descriptions":[{"lang":"en","value":"CUPS is a standards-based, open-source printing system, and cups-filters provides backends, filters, and other software for CUPS 2.x to use on non-Mac OS systems. Any value passed to `FoomaticRIPCommandLine` via a PPD file will be executed as a user controlled command. When combined with other logic bugs as described in CVE_2024-47176, this can lead to remote command execution."},{"lang":"es","value":"CUPS es un sistema de impresión de código abierto basado en estándares, y cups-filters proporciona backends, filtros y otro software para que CUPS 2.x se utilice en sistemas que no sean Mac OS. Cualquier valor que se pase a `FoomaticRIPCommandLine` a través de un archivo PPD se ejecutará como un comando controlado por el usuario. Cuando se combina con otros errores lógicos como los descritos en CVE_2024-47176, esto puede provocar la ejecución remota de comandos."}],"metrics":{"cvssMetricV31":[{"source":"security-advisories@github.com","type":"Secondary","cvssData":{"version":"3.1","vectorString":"CVSS:3.1\/AV:N\/AC:H\/PR:N\/UI:N\/S:C\/C:H\/I:H\/A:H","attackVector":"NETWORK","attackComplexity":"HIGH","privilegesRequired":"NONE","userInteraction":"NONE","scope":"CHANGED","confidentialityImpact":"HIGH","integrityImpact":"HIGH","availabilityImpact":"HIGH","baseScore":9.0,"baseSeverity":"CRITICAL"},"exploitabilityScore":2.2,"impactScore":6.0}]},"weaknesses":[{"source":"security-advisories@github.com","type":"Primary","description":[{"lang":"en","value":"CWE-77"}]}],"references":[{"url":"https:\/\/github.com\/OpenPrinting\/cups-browsed\/security\/advisories\/GHSA-rj88-6mr5-rcw8","source":"security-advisories@github.com"},{"url":"https:\/\/github.com\/OpenPrinting\/cups-filters\/security\/advisories\/GHSA-p9rh-jxmq-gq47","source":"security-advisories@github.com"},{"url":"https:\/\/github.com\/OpenPrinting\/libcupsfilters\/security\/advisories\/GHSA-w63j-6g73-wmg5","source":"security-advisories@github.com"},{"url":"https:\/\/github.com\/OpenPrinting\/libppd\/security\/advisories\/GHSA-7xfx-47qg-grp6","source":"security-advisories@github.com"},{"url":"https:\/\/www.cups.org","source":"security-advisories@github.com"},{"url":"https:\/\/www.evilsocket.net\/2024\/09\/26\/Attacking-UNIX-systems-via-CUPS-Part-I","source":"security-advisories@github.com"}]}}]}
```

Comment:
Can you format this JSON file to make it easier to read?

Reply:
Updated format of all nvdcve-2.0 json files